app/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 android {
-    compileSdk 30
+    compileSdk 31
```

Comment:
there is nothing technically stopping you from having a target and compile SDK that differ, but generally you'll want them to be the same

Reply:
Yeah, that has been a point of frustration: I will see if I can increase the targetSDK to 31. I can't remember why I initially had to reduce it. Thanks for the feedback!